environments/azure/sandbox/cloudprovider/terragrunt.hcl

```diff
@@ -0,0 +1,12 @@
+terraform {
+    source = "../../../../../stacks/environment-azure"
```

Comment:
I think this is too many levels back now

Reply:
good catch...fixed